flask_app/app.py

```diff
@@ -10,7 +10,7 @@
 import traceback
 from tensorflow.keras.models import load_model
 import numpy as np
-
+import os
```

Comment:
This was added when trying different filepath options. We can remove this, but it is not harming anything, I just forgot about it